### PR TITLE
Added package level docs

### PR DESCRIPTION
**Pull request by ewoutp**
This PR extends the docs are they are shown on godoc.org.

### CODE REVIEW

File: http/doc.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+//
+// Copyright holder is ArangoDB GmbH, Cologne, Germany
+//
+// Author Ewout Prangsma
+//
+
+/*
+Package http implements driver.Connection using an HTTP connection.
+
+This connection uses HTTP or HTTPS to connect to the ArangoDB databaseand
```
Comment (neunhoef) [inline http/doc.go]: Space missing: `databaseand`

File: http/doc.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+	c, err := driver.NewClient(driver.ClientConfig{
+		Connection: conn,
+	})
+	if err != nil {
+		// Handle error
+	}
+
+Creating a Secure Connection
+
+To create an secure HTTPS connection, use code like this.
```
Comment (neunhoef) [inline http/doc.go]: a secure

File: vst/doc.go [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+	c, err := driver.NewClient(driver.ClientConfig{
+		Connection: conn,
+	})
+	if err != nil {
+		// Handle error
+	}
+
+Creating a Secure Connection
+
+To create an secure VST connection, use code like this.
```
Comment (neunhoef) [inline vst/doc.go]: a secure

**Comment by ewoutp** [comment]
Typos fixed

**Pull request merged**